ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/normalization.py

```diff
@@ -24,8 +24,7 @@ def normalize_data(df, **kwargs):
     scaler_type = kwargs["norm"]
     columns = kwargs["columns"]
 
-    columns = columns[0].split(
-        " ") if len(columns) == 1 else columns
+    columns = columns.split()
```

Comment:
[Minor] Please give this another thought because I think currently there is no restriction on column names. So, this could potentially generate column names (splitted by default space charactere) which are not available in training/inference data. Similar comment on `groupby` as well.

Reply:
That's a good point, but this is a common issue existing in all of our transformer code for now. I only make this change to fix problem that will break our workflow when users are using our solution correctly.

I think it's worthwhile to open a separate ticket for tracking all user input validation in our code base, not only transformer

Reply:
That's a good point. Sure, sounds good, thanks :)